src/Microsoft.ML.OnnxTransformer/Microsoft.ML.OnnxTransformer.csproj

```diff
@@ -9,6 +9,7 @@
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.ML.Core\Microsoft.ML.Core.csproj" />
     <ProjectReference Include="..\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
+    <ProjectReference Include="..\Microsoft.ML.OnnxConverter\Microsoft.ML.OnnxConverter.csproj" />
```

Comment:
You can't make this change without also adding the reference to the NuGet package.

But in general, I don't think making OnnxTransformer dependent on OnnxConverter is a good idea. Instead, if there is code that needs to be shared, we can compile it into both libraries.

Reply:
Sure. I will compile the required part into ONNXTransformer instead of reusing it from ONNXConverter.